### ISSUE TITLE
Skip needs polishing

### ISSUE THREAD

**Issue by Nivrad00** [Nivrad00/social-landmines#3]
A few issues: 
1) you can rollback while skipping and it leads to weird bugs (fixed for now)
2) it's hard to tell when you're skipping or not bc the only indication is the "skip" button being highlighted
3) there's no way to delete your data, so once a line gets marked as "seen" it's forever "seen" (this also makes it hard to test if the "skip unseen" toggle works correctly or not)

**Comment by Nivrad00** [comment]
Oh
4. "after choices" toggle doesn't seem to do anything

**Comment by Nivrad00** [comment]
I removed the "skip unseen" and "after choices" toggles for now (as well as "auto forward speed," which i believe isn't related to skipping but has to do with a feature that automatically advances the story at regular intervals?) they don't seem to be implemented well, and even if they are, we don't really need those features.

**Comment by Nivrad00** [comment]
not sure if it skips unseen text or not

**Comment by Nivrad00** [comment]
Okay, I think I resolved all the issues with skipping, namely:
1. Rollback is now disabled while skipping
2. Added an additional GUI element that pops up when you're skipping to make it more obvious
3. Removed feature of only skipping "seen" steps, due to general bugginess -- now it skips everything but choices (and inputs? haven't tested)
4. Removed settings corresponding to "skip unseen" and "skip after choices" because they weren't fully implemented
5. If you went to the main menu while skipping, it would continue skipping after you start a new game -- so I fixed that too
